### PR TITLE
productListing page

### PR DESCRIPTION
**Pull request by harshalini**
Added product-listing page for vivir-fashion
The product listing pages has card for each product with -
1) Product Image
2) Product name
3) Product description
4) Product discounted price and original price
5) Add to wishlist button
6) Add to card button
Screenshot (438)
Check the preview here ->
https://6236181248f3535eafbd6aef--gracious-hodgkin-180ba1.netlify.app/products

### CODE REVIEW

**Review by SuyashTiwariCoder — APPROVED** [review]
I like your clean code  , good formatting , Approved you can merge .

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+import { ProductsPage } from "./pages/productsPage/products";
 import Mockman from "mockman-js"
 function App() {
   return (
     <div className="App">
-      <Home />
+      <Routes>
+        <Route exact path = "/" element = {<Home/>} />
+        <Route path = "products" element = {<ProductsPage/>} />
+      </Routes>
```
Comment (dikshadhote) [inline src/App.js]: Can create separate file for routes and then import it in App

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom"
 import { Home } from "./pages/home/home"
+import { ProductsPage } from "./pages/productsPage/products";
```
Comment (dikshadhote) [inline src/App.js]: You can import all components in one file and then use from that file

File: src/pages/productsPage/products.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Navbar } from "../../components/homeComp"
+import { GetProducts } from "../../components/products/product-card"
```
Comment (dikshadhote) [inline src/pages/productsPage/products.js]: same as above

**Review by dikshadhote — APPROVED** [review]
Overall the code is neat and clean. But you can make it modular by adding route and component file as separate file

File: src/App.js (hunk 1 above)
Comment (harshalini) [reply src/App.js]: Ok, thank you for suggestion!

**Comment by harshalini** [comment]
> I like your clean code , good formatting , Approved you can merge .

Thank you

**Pull request merged**